src/backends/posix2/run.c

```diff
@@ -310,6 +311,8 @@ int psh_backend_do_run(psh_state *state, struct _psh_command *cmd)
             printf("argv[%d] = %s\n", j, cmd->argv[j]);
         printf("flag: %d\n", cmd->type);
 #endif
+        /* Replace command with alias */
+        replace_with_alias(cmd);
```

Comment:
I'm not sure whether `bash` expands aliases before or after word splitting, is it possible to check that only through `bash`'s behavior? If not maybe I'll check the source code.